packages/im-components/src/molecules/logo/logo.module.scss

```diff
@@ -0,0 +1,22 @@
+.image {
+  display: inline-block;
+  height: 100%;
+  padding: 0rem 2rem;
+}
+
+.logo {
+  display: inline-flex !important;
+  color: #000;
+  align-items: center;
+}
+
+.logo:hover {
+  color: #000
+}
+
```

Comment:
same comments like before

Reply:
Will pull changes from develop once #18  is merged. updated it there.